examples/text_classification/models/estimator/src/func_main.py

```diff
@@ -0,0 +1,27 @@
+from keras.models import load_model
+import hydro_serving_grpc as hs
+import tensorflow as tf
+import numpy as np
+amazon_model = load_model('/model/files/amazon_model.h5')
+amazon_model._make_predict_function()
+graph = tf.get_default_graph()
+def predict(tokenized):
+    global graph
+    tokenized_sentence = tokenized.int64_val
+    tokenized_sentence = np.array([tokenized_sentence])
```

Comment:
You can add `ndmin=2` to `np.array` call to construct 2D array with batch_dim=1, so you dont need to pass `tokenzied_sentence` in brackets in line 13